src/components/settings/SettingData.jsx

```diff
@@ -10,23 +10,31 @@ class SettingData extends React.Component {
     super(props);
 
     this.state = {
-      alert: false
+      showAlert: false
     };
   }
 
-  handleAlert = () => {
-    this.setState({ alert: true });
+  handleShowAlert = () => {
+    this.setState({ showAlert: true });
+  }
+
+  handleCloseAlert = () => {
+    this.setState({ showAlert: false });
   }
 
   render() {
     const { optionName } = this.props;
-    const { alert } = this.state;
+    const { showAlert } = this.state;
 
     return (
-      <div style={{ float: 'left' }}>
-        {`${optionName} `}
-        <CloseIcon style={{ cursor: 'pointer' }} onClick={this.handleAlert} />
-        {alert && (<AlertDialog optionName={optionName} />)}
+      <div style={{ width: '100%', marginTop: '5px', marginBottom: '5px' }}>
+        <div style={{ clear: 'both', float: 'left', width: 'auto' }}>
+          {`${optionName} `}
+        </div>
+        <div style={{ float: 'right', width: 'auto' }}>
+          <CancelIcon fontSize="inherit" style={{ cursor: 'pointer', color: '#99999', verticalAlign: 'middle' }} onClick={this.handleShowAlert} />
+          <AlertDialog showAlert={showAlert} {...this.props} onClose={this.handleCloseAlert} />
```

Comment:
Can we just pass optionName explicitly rather than destructure an entire props object should the object change in the future.